butane_core/src/db/pg.rs

```diff
@@ -551,9 +551,22 @@ fn sql_for_op(current: &mut ADB, op: &Operation) -> Result<String> {
         Operation::AddTableConstraints(table) => Ok(create_table_constraints(table)),
         Operation::AddTableIfNotExists(table) => Ok(create_table(table, true)?),
         Operation::RemoveTable(name) => Ok(drop_table(name)),
+        Operation::RemoveTableConstraints(table) => remove_table_constraints(table),
```

Comment:
Why does this need to be a separate operation semantically? Why not just do it automatically as part of `RemoveTable` handling? It looks like ADB `diff` is always pairing them anyway, and I would argue that whether a table's constraints have to be dropped before the table can be dropped is a backend-specific attribute. Theoretically there could be a backend for which that wasn't true.

Reply:
Would have to change the signature of `RemoveTable` slightly, but I think that's OK

Reply:
We need to process all of the tables twice - one to drop their fkey constraints , and then the tables can be dropped.